### ISSUE TITLE
More details regarding Places-LT

### ISSUE THREAD

**Issue by TonyLianLong** [facebookresearch/classifier-balancing#4]
Hi @bingykang , I have a few questions regarding the project. Since you haven't yet released the stage 1 config for Places-LT, I would like to know the following details:

1. How many epochs did you train Places-LT for in the first stage?
2. What is frozen and what is unfrozen in that stage?
3. What learning rate scheduler do you use, and what about the parameters?
4. Are there any changes that needs to be done in your code before fine-tuning from the ImageNet pretrain?

Thanks for answering my Qs, and have a nice day!

**Comment by bingykang** [comment]
1. It's either 30 or 90 epochs. 
2. Everything is frozen except for the last block and fc layer. 
3. A cosine learning rate schedule is used as other datasets. 
4. Follow the explanation here: https://github.com/facebookresearch/classifier-balancing/issues/2

**Comment by TonyLianLong** [comment]
Thanks!
